src/main/kotlin/uk/gov/dluhc/printapi/service/StatisticsUpdateService.kt

```diff
@@ -3,29 +3,15 @@ package uk.gov.dluhc.printapi.service
 import org.springframework.stereotype.Service
 import uk.gov.dluhc.messagingsupport.MessageQueue
 import uk.gov.dluhc.votercardapplicationsapi.messaging.models.UpdateApplicationStatisticsMessage
-import uk.gov.dluhc.votercardapplicationsapi.messaging.models.UpdateStatisticsMessage
 import java.util.UUID
 
 @Service
 class StatisticsUpdateService(
-    private val triggerVoterCardStatisticsUpdateQueue: MessageQueue<UpdateStatisticsMessage>,
     private val triggerApplicationStatisticsUpdateQueue: MessageQueue<UpdateApplicationStatisticsMessage>,
 ) {
 
-    fun updateStatistics(applicationId: String, isFromApplicationsApi: Boolean? = null) {
-        if (isFromApplicationsApi == true)
-            triggerApplicationStatisticsUpdate(applicationId)
-        else
-            triggerVoterCardStatisticsUpdate(applicationId)
-    }
-    fun triggerVoterCardStatisticsUpdate(applicationId: String) {
-        triggerVoterCardStatisticsUpdateQueue.submit(
-            UpdateStatisticsMessage(voterCardApplicationId = applicationId),
-            mapOf(
-                "message-group-id" to applicationId,
-                "message-deduplication-id" to UUID.randomUUID().toString(),
-            )
-        )
+    fun updateStatistics(applicationId: String) {
+        triggerApplicationStatisticsUpdate(applicationId)
     }
```

Comment:
one of these functions should now be replaced as one just calls the other.

Do we have a naming preference to keep one or the other? longer one is more specific, but probably was also named like that to differentiate it from a function that doesn't exist any more